Dockerfile.builder

```
@@ -17,8 +17,8 @@ RUN go get -u golang.org/x/lint/golint
 RUN go get github.com/jstemmer/go-junit-report
 
 ### Install NodeJS and yarn
-ENV NODE_VERSION="v10.17.0"
-ENV YARN_VERSION="v1.7.0"
+ENV NODE_VERSION="v14.15.0"
```

Let's go ahead and use 

```suggestion
ENV NODE_VERSION="v14.16.0"
```

---

Actuallly, 14.15.4 might be better to align with what ART will use, and 14.15 is LTS

```suggestion
ENV NODE_VERSION="v14.15.4"
```

---

It turns out RHEL 8 just bumped to 14.16.0 for several CVEs, so we probably need to retarget that.

---

I'm good with 14.16.0.